### PR TITLE
v1.27 added link handling to webviews

### PR DESCRIPTION
**Pull request by gokmev**
Handling mail , phone and direction links in webviews
- Updated sample app to sow webviews
- Updated kotlin version

### CODE REVIEW

File: sample/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 		  package="com.lush.views.sample">
 
+	<uses-permission android:name="android.permission.INTERNET"/>
```
Comment (JamieCruwys) [inline sample/src/main/AndroidManifest.xml]: Why do we need the internet permission?

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+/**
+ * Handler class to handle telephone, navigation and e-mail links
+ *
+ * @author Gokhan
+ */
+class LinkHandler
+{
+	companion object
+	{
+		@JvmStatic
```
Comment (JamieCruwys) [inline views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: Please remove JVMStatic and companion object uses

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+					{
+						val intent = Intent(Intent.ACTION_DIAL, Uri.parse(urlString))
+						attemptStartActivity(context, intent)
+						return true
+					}
+					urlString.startsWith("mailto:") ->
+					{
+						val url = urlString.substring(7)
+						val mail = Intent(Intent.ACTION_SEND)
+						mail.type = "application/octet-stream"
```
Comment (JamieCruwys) [inline views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: Why an octet-stream?

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+
+/**
+ * Handler class to handle telephone, navigation and e-mail links
+ *
+ * @author Gokhan
+ */
+class LinkHandler
```
Comment (JamieCruwys) [inline views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: Nice abstraction of functionality 👌

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt (hunk 2 above)
Comment (JamieCruwys) [reply views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: Once you remove both, you can access methods by calling `LinkHandler.INSTANCE.handleLink()`

File: sample/src/main/AndroidManifest.xml (hunk 1 above)
Comment (gokmev) [reply sample/src/main/AndroidManifest.xml]: This is to test the webview with a url in sample app

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt (hunk 3 above)
Comment (gokmev) [reply views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: changing to plain text

File: views/src/main/java/com/lush/lib/handler/LinkHandler.kt (hunk 4 above)
Comment (gokmev) [reply views/src/main/java/com/lush/lib/handler/LinkHandler.kt]: Thanks :)

**Pull request merged**